optimum/exporters/onnx/model_configs.py

```diff
@@ -610,6 +610,10 @@ class YolosOnnxConfig(ViTOnnxConfig):
     DEFAULT_ONNX_OPSET = 12
 
 
+class SwinOnnxConfig(ViTOnnxConfig):
+    DEFAULT_ONNX_OPSET = 12
```

Comment:
Is it needed for sure? Just checking

Reply:
Right, goes well with opset = 11 as well!